app/ide-desktop/lib/dashboard/e2e/api.ts

```diff
@@ -350,23 +349,23 @@ export async function mockApi({ page }: MockParams) {
         const projectId = request.url().match(/[/]projects[/](.+?)[/]copy/)?.[1] ?? ''
         await route.fulfill({
           json: {
-            /* eslint-disable @typescript-eslint/naming-convention */
             organizationId: defaultOrganizationId,
             projectId: backend.ProjectId(projectId),
             name: 'example project name',
             state: {
               type: backend.ProjectState.opened,
-              volume_id: '',
-              opened_by: defaultEmail,
+              volumeId: '',
+              openedBy: defaultEmail,
             },
             packageName: 'Project_root',
+            /* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
prefer two separet `eslint-disable-next-line`s, that way we don't risk forgetting to re-enable it